azurerm/internal/services/network/firewall_policy_data_source.go

```diff
@@ -63,10 +63,6 @@ func dataSourceArmFirewallPolicy() *schema.Resource {
 							Type:     schema.TypeBool,
 							Computed: true,
 						},
-						"network_rule_fqdn_enabled": {
```

Comment:
instead of removing this could we just deprecate it and set it to a zero value? otherwise people might not know why the property dissapeared

Reply:
Sure, we can set it to a zero value, however, since this is a Computed property, I don't think "Deprecated" would work for this.

Reply:
Have you checked? I think if you are using a deprecated datasource property it will still give a deprecation notice.. but i could be misremembering

Reply:
I think it doesn't matter whether the property is from data source or resource, but the point is that it is a `Computed` (only) one.

Given following schema:

```go
...
"output_job": {                   
    Type:       schema.TypeString,
    Deprecated: "dubaduba..",     
    Computed:   true,             
},
...                                
```

and following config:

```hcl
resource "demo_resource_foo" "magodo" {
  name = "magodo"
  job = "dev"
}
resource "demo_resource_foo" "kinoko" {
  name = "kinoko"
  job = demo_resource_foo.magodo.output_job
}
```

I got:

```bash
💤 tf apply 

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # demo_resource_foo.kinoko will be created
  + resource "demo_resource_foo" "kinoko" {
      + id             = (known after apply)
      + job            = (known after apply)
      + name           = "kinoko"
      + output_addr    = (known after apply)
      + output_contact = (known after apply)
      + output_job     = (known after apply)
    }

  # demo_resource_foo.magodo will be created
  + resource "demo_resource_foo" "magodo" {
      + id             = (known after apply)
      + job            = "dev"
      + name           = "magodo"
      + output_addr    = (known after apply)
      + output_contact = (known after apply)
      + output_job     = (known after apply)
    }

Plan: 2 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

demo_resource_foo.magodo: Creating...
demo_resource_foo.magodo: Creation complete after 0s [id=magodo]
demo_resource_foo.kinoko: Creating...
demo_resource_foo.kinoko: Creation complete after 0s [id=kinoko]

Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
```

Reply:
Well that's likely a disappointing change from how things used to work - thanks for checking!